README.md

```diff
@@ -53,113 +53,132 @@ Follow each of the steps below to install automation-tools:
+    sudo chown $USER /usr/lib/archivematica/automation-tools &&
+    sudo chown $USER /usr/share/python/automation-tools/ &&
+    sudo chown $USER /etc/archivematica/automation-tools/
+    ```
+
+    In case of using the automation tools with a Docker setup the archivematica 
+    user and group don't exist:
+
+    ```
+    sudo useradd archivematica
+    ```
 
 3. Clone the automation-tools repository into your `usr/lib` directory:
 
-```bash
-cd /usr/lib/archivematica/automation-tools/
-git clone https://github.com/artefactual/automation-tools.git .
-```
+    ```
+    cd /usr/lib/archivematica/automation-tools/
+    git clone https://github.com/artefactual/automation-tools.git .
+    ```
+
+    In case you want to run the automation-tools using [cron](#configuration], the
+    automation tools need to be owned by the archivemtatica user:
+    
+    ```
+    sudo chown archivematica:archivematica /usr/lib/archivematica/automation-tools/
+    ```
 
 4. Set up a Python virtual environment:
 
-```bash
-cd /usr/share/python/automation-tools
-virtualenv venv
-source venv/bin/activate
-pip install -r /usr/lib/archivematica/automation-tools/requirements.txt
-```
+    ```
+    cd /usr/share/python/automation-tools
+    virtualenv venv
+    source venv/bin/activate
+    pip install -r /usr/lib/archivematica/automation-tools/requirements.txt
+    ```
 
 5. (Optional) Update and retrieve required packages:
 
-```bash
-apt-get update
-apt-get install p7zip-full
-```
+    ```
+    apt-get update
+    apt-get install p7zip-full
+    ```
 
+Usage
+-----
 To use automation-tools, copy the scripts you want to use into your
 `/etc/archivematica/automation-tools` directory.
 
+`transfers/transfer.py` is used to prepare transfers, move them into the
+pipelines processing location, and take actions when user input is required.
+Only one transfer is sent to the pipeline at a time, the scripts wait until the
+current transfer is resolved (failed, rejected or stored as an AIP) before
+automatically starting the next available transfer.
+
 Follow the steps below as an example for setting up a script.
 
 1. Copy the configuration file:
 
-`cp /usr/lib/archivematica/automation-tools/etc/transfers.conf
-/etc/archivematica/automation-tools/`
+    ```
+    `cp /usr/lib/archivematica/automation-tools/etc/transfers.conf
+    /etc/archivematica/automation-tools/`
+    ```
 
 2. Copy the script:
 
-`cp /usr/lib/archivematica/automation-tools/etc/transfer-script.sh
-/etc/archivematica/automation-tools/`
+    ```
+    cp /usr/lib/archivematica/automation-tools/etc/transfer-script.sh /etc/archivematica/automation-tools/
+    ```
 
 3. Create a transfer source for the automation-tools in the storage service:
 
-You can use an existing transfer source, or set up a new one.
+    You can use an existing transfer source, or set up a new one. For Docker 
+    environments the standard transfer source is `$HOME/.am/ss-location-data/`.
```

Comment:
We could highlight that if folks are running docker as `sudo` they might also need to modify the ownership of this folder.

Reply:
Yest will do that.